asset-services/asset-services-celery/src/preflight/types.rs

```diff
@@ -0,0 +1,63 @@
+//! Preflight check level type.
+
+use std::str::FromStr;
+
+use env_var_helpers::env_vars;
+use strum::EnumString;
+
+/// Specify what level of preflight checks to run (inclusive).
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Debug)] // core
+#[derive(EnumString)] // strum
+#[strum(ascii_case_insensitive)]
+#[strum(serialize_all = "snake_case")]
+pub enum CheckLevel {
```

Comment:
This representation does not seem super clear from just looking at the code and types.

Three options I see that might make it clearer:
 1. Since we are comparing them similar to numeric values in the `impl`, maybe assign a spesific numberic value to the enum values
 2. Use a bitflags style enum (probably overkill for this)
 3. Name the values `None, Local, All` instead. - (I think I prefer this)


This is not a major issue, since the comments do clarify the intent, just came up during review.